manifests/config.pp

```diff
@@ -0,0 +1,169 @@
+        source => "puppet:///modules/nzbget/${nzbget::unpack_pass_file_name}",
+      }
+    }
+  }
+
+  file { $nzbget::log_file:
+    ensure => present,
+    owner  => $::nzbget::user,
+    group  => $::nzbget::group,
+    mode   => '0700',
+  }
+
+  file { $nzbget::lock_file:
+    ensure => present,
+    owner  => $::nzbget::user,
+    group  => $::nzbget::group,
+    mode   => '0700',
+  }
+
+  concat { $nzbget::config_file:
+    ensure => 'present',
+    owner  => $::nzbget::user,
+    group  => $::nzbget::group,
+    mode   => '0640',
+  }
+
+  concat::fragment { 'head':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_head.erb'),
+    order   => '01',
+  }
+
+  concat::fragment { 'paths':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_paths.erb'),
+    order   => '03',
+  }
+
+  concat::fragment { 'servers':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_servers.erb'),
+    order   => '05',
+  }
+
+  concat::fragment { 'security':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_security.erb'),
+    order   => '07',
+  }
+
+  concat::fragment { 'categories':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_categories.erb'),
+    order   => '09',
+  }
+
+  concat::fragment { 'rss_feeds':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_rss_feeds.erb'),
+    order   => '11',
+  }
+
+  concat::fragment { 'incoming':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_incoming.erb'),
+    order   => '13',
+  }
+
+  concat::fragment { 'download':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_download.erb'),
+    order   => '15',
+  }
+
+  concat::fragment { 'connection':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_connection.erb'),
+    order   => '17',
+  }
+
+  concat::fragment { 'logging':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_logging.erb'),
+    order   => '19',
+  }
+
+  concat::fragment { 'display':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_display.erb'),
+    order   => '21',
+  }
+
+  concat::fragment { 'scheduler':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_scheduler.erb'),
+    order   => '23',
+  }
+
+  concat::fragment { 'check_and_repair':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_check_and_repair.erb'),
+    order   => '25',
+  }
+
+  concat::fragment { 'unpack':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_unpack.erb'),
+    order   => '27',
+  }
+
+  concat::fragment { 'extension':
+    target  => $nzbget::config_file,
+    content => template('nzbget/nzbget.conf_extension.erb'),
+    order   => '29',
```

Comment:
Does the order of the config file really matter? If not, let's just get rid of the ordering options.